release/build_release.sh

```diff
@@ -26,6 +26,16 @@ function check_usage {
 		print_help
 	fi
 
+	if [ -z "${LIFERAY_RELEASE_SOFT}" ]
+	then
+		LIFERAY_RELEASE_SOFT="false"
+	fi
+
+	if [ "${LIFERAY_RELEASE_SOFT}" == "true" ] && { [ -z "${LIFERAY_RELEASE_TICKETS}" ] || [ -z "${LIFERAY_RELEASE_GIT_PREV_REF}" ]; }
```

Comment:
@kiwm, do we need the `;`?

Reply:
I didn't think so either, but if I don't put it, the IDE shows those errors.
Expected a '}'. If you have one, try a ; or \n in front of it.
See SC1056.
Couldn't parse this brace group. Fix to allow more checks.
See SC1073.